main.go

```diff
@@ -229,6 +233,7 @@ func main() {
 	restConfig := ctrl.GetConfigOrDie()
 	restConfig.QPS = restConfigQPS
 	restConfig.Burst = restConfigBurst
+	restConfig.Timeout = drainKubeTimeout
```

Comment:
I assume the idea was to set this on the machine Reconciler instead, right?

Reply:
Whoops! A leftover line copied over from my previous PR. Fixed.